languagetool-language-modules/de/src/main/resources/org/languagetool/resource/de/added.txt

```diff
@@ -37324,4 +37324,12 @@ Mythosschwarzs	Mythosschwarz	SUB:GEN:SIN:NEU
 Felsgrau	Felsgrau	SUB:AKK:SIN:NEU
 Felsgrau	Felsgrau	SUB:DAT:SIN:NEU
 Felsgrau	Felsgrau	SUB:NOM:SIN:NEU
-Felsgraus	Felsgrau	SUB:GEN:SIN:NEU
\ No newline at end of file
+Felsgraus	Felsgrau	SUB:GEN:SIN:NEU
+Fantasyserie    Fantasyserie  SUB:AKK:SIN:FEM
```

Comment:
You need to use tabs here, not spaces.

Reply:
Ah, sorry, seems like my editor converted the tabs to spaces. Will fix tonight.

Reply:
should be fixed now